assets/js-templates/form-components.php

```diff
@@ -113,9 +113,16 @@ class="wpuf-flex wpuf-justify-between wpuf-p-6 wpuf-rounded-t-md wpuf-border-t w
                     <label
                         v-if="!is_invisible(field)"
                         :for="'wpuf-' + field.name ? field.name : 'cls'"
-                        class="wpuf-block wpuf-text-sm wpuf-font-medium wpuf-leading-6 wpuf-text-gray-900">
+                        class="wpuf-flex wpuf-items-center wpuf-text-sm wpuf-font-medium wpuf-leading-6 wpuf-text-gray-900">
                         {{ field.label }} <span v-if="field.required && 'yes' === field.required"
                                                 class="required">*</span>
+                        <template v-if="field.show_icon === 'yes' && field.template && field.template.endsWith('_url')">
```

Comment:
is there any other way to check instead of `field.template.endsWith('_url')`? why are we doing this check `field.template.endsWith('_url')`?